pkg/operation/botanist/component/extensions/worker/dependency_watchdog.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package worker
+
+import (
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+
+	proberapi "github.com/gardener/dependency-watchdog/api/prober"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	"k8s.io/utils/pointer"
+)
+
+// DependencyWatchdogProbeConfiguration returns the configuration for the dependency watchdog (probe role)
+// ensuring that its dependant pods are scaled as soon a probe fails.
+func DependencyWatchdogProbeConfiguration() ([]proberapi.DependentResourceInfo, error) {
+	return []proberapi.DependentResourceInfo{
+		{
+			Ref: &autoscalingv1.CrossVersionObjectReference{
+				Kind:       "Deployment",
+				Name:       v1beta1constants.DeploymentNameClusterAutoscaler,
+				APIVersion: "apps/v1",
+			},
+			ShouldExist: pointer.Bool(false),
+			ScaleUpInfo: &proberapi.ScaleInfo{
+				Level:    2,
+				Replicas: pointer.Int32(1),
```

Comment:
Just add a note that in case the number of replicas change to a value > 1 then DWD maintainers should be notified.